setup.py

```diff
@@ -10,7 +10,7 @@
     description="Zyte's Page Object pattern for web scraping",
     long_description=long_description,
     long_description_content_type="text/x-rst",
-    author="Scrapinghub",
+    author="Zyte Group Ltd",
     author_email="info@scrapinghub.com",
```

Comment:
We could update the email as well to `info@zyte.com` to be in line with the doc changes.

Reply:
@Gallaecio could you please advice us here? When doing the rebranding, what was the email you were using?

Reply:
For open source projects I use opensource@zyte.com.